Soomla/Assets/Plugins/Soomla/Profile/social-providers/FBSocialProvider.cs

```diff
@@ -150,7 +150,32 @@ public FBSocialProvider ()
             });
         }
         
-        /// <summary>
+
+		/// <summary>
+		/// See docs in <see cref="SoomlaProfile.UpdateStatus"/>
+		/// </summary>
+		/// <param name="link">Link to post.</param>
+		/// <param name="success">Callback function that is called if the status update was successful.</param>
+		/// <param name="fail">Callback function that is called if the status update failed.</param>
+		public override void UpdateStatusDialog(string link, SocialActionSuccess success, SocialActionFailed fail) {
+			FB.Feed(
+				link: link,
+				callback: (FBResult result) => {
+				
+				if (result.Error != null) {
+					fail(result.Error);
+				}
+				else {
+					SoomlaUtils.LogDebug(TAG, "FeedCallback[result.Text]:"+result.Text);
+					SoomlaUtils.LogDebug(TAG, "FeedCallback[result.Texture]:"+result.Texture);
```

Comment:
@vedi, according to Android implementation of Facebook provider, `updateStory` and `updateStatus` has the following end-point invocation:
`SimpleFacebook.getInstance().publish(feed, false, new OnPublishListener() {`
The difference is in number of parameters (status update has only message, and story update has several parameters).